verl/trainer/config/rollout/rollout.yaml

```diff
@@ -1,5 +1,5 @@
 # actor_rollout_ref.rollout.name: hf/vllm/sglang.
-name: vllm
+name: ???
```

Comment:
Should we add a sanity check in `_validate_config`?

Reply:
??? is a missing config. If this field is not set, and the field is accessed, a MISSING field exception will be raised.